geode-core/src/main/java/org/apache/geode/pdx/internal/PeerTypeRegistration.java

```diff
@@ -786,20 +763,11 @@ public PdxType getPdxTypeForField(String fieldName, String className) {
   /**
    * For testing purpose
    */
+  @Deprecated
```

Comment:
We have a `@TestingOnly` annotation that might fit better than `@Deprecated` here, given that it's only used in tests and is internal (so deprecation is only kind of a thing anyway).